.lintr

```diff
@@ -1,4 +1,4 @@
-linters: with_defaults(
+linters: linters_with_defaults(
   line_length_linter = line_length_linter(120),
```

Comment:
I believe these specs are also changed. It becomes a group of args instead of a list.

Reply:
I think it is ok 
Examples: https://lintr.r-lib.org/articles/lintr.html#customizing-active-linters

Reply:
Cool